composer.json

```diff
@@ -2,17 +2,17 @@
   "name": "lerouse/laravel-change-control",
   "description": "Laravel change control/version management package.",
   "keywords": ["laravel", "changelog", "version", "control"],
-  "type": "package",
+  "type": "library",
```

Comment:
I did this as "library" is the name used by the composer.json spec: https://getcomposer.org/doc/04-schema.md#type